renku/cli/run.py

```diff
@@ -15,7 +15,63 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Track provenance of data created by executing programs."""
+"""Track provenance of data created by executing programs.
+
+Capture command line execution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tracking exection of your command line script is done by simply adding
+``renku run`` command before the previous arguments. The command will detect
+
+* arguments (flags),
+* string and integer options,
+* input files if linked to existing files in the repository,
```

Comment:
should we mention that inputs need to be specified on the command line? Should be obvious but maybe it isn't?

Reply:
I have tried to put more details in `Detecting input files`